collect/cache/cache.go

```diff
@@ -90,18 +90,18 @@ func (d *DefaultInMemCache) GetCacheEntryCount() int {
 	return len(d.cache)
 }
 
-func (d *DefaultInMemCache) Set(trace *types.Trace) *types.Trace {
+func (d *DefaultInMemCache) Set(trace *types.Trace) {
 	// we need to dereference the trace ID so skip bad inserts to avoid panic
 	if trace == nil {
-		return nil
+		return
 	}
 	start := time.Now()
 
-	defer d.Metrics.Histogram("trace_cache_set_dur_ms", time.Since(start).Milliseconds())
+	defer d.Metrics.Histogram("trace_cache_set_dur_ms", float64(time.Since(start).Milliseconds()))
```

Comment:
I'm curious why you needed to do this. Milliseconds() returns an integer, and the metrics code converts everything to floats anyway.

Reply:
Hmm, I thought that's the reason why it's reporting 0 value right now. Is it just that fast???
https://ui.honeycomb.io/kibble/environments/kibble/datasets/refinery-metrics/result/gwPNGa96CNB